website/main.py

```diff
@@ -1,10 +1,14 @@
 from starlette.applications import Starlette
+from starlette.responses import JSONResponse, RedirectResponse
 from starlette.routing import Mount, Request, Route
 from starlette.staticfiles import StaticFiles
 from starlette.templating import Jinja2Templates
+from starlette_discord.client import DiscordOAuthClient
 
+from website.constants import CLIENT_ID, CLIENT_SECRET, REDIRECT_URI
 
 templates = Jinja2Templates(directory="website/templates")
+client = DiscordOAuthClient(CLIENT_ID, CLIENT_SECRET, REDIRECT_URI)
```

Comment:
I'm not sure that `client` is the best variable name here. Perhaps something like `discord_client`?